sdk/template/azure-template/CHANGELOG.md

```diff
@@ -99,3 +73,32 @@
 ## 1.0.0-beta.1 (2020-09-01)
 
 * Template package validating release pipeline
+## 1.0.0-beta.25 (2020-01-12)
```

Comment:
@danieljurek Why did these get moved out of order?

Reply:
We recently changed the sorting to be based on release date first then version and it looks like these entries don't have the correct dates. Seems as though they should be 2021 for the year.

Reply:
I just merged a [PR](https://github.com/Azure/azure-sdk-for-cpp/pull/1936/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml) that fixes up the dates. This should be closed anyways.